README.md

```diff
@@ -26,7 +26,7 @@ Docker version 19.03.1, build 74b1e89
 root@test-68b8628f:~# docker-compose --version
 docker-compose version 1.22.0, build f46880fe
 ```
-* Hardware: 2 CPUs, 8GB memory RAM and [enough virtual memory for Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html) 
+* **Hardware**: 2 CPUs, 8GB memory RAM and [enough virtual memory for Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html) 
```

Comment:
This is not really related to the highlighting, but just in case: do we really need 2 CPUs? Isn't it more like 2 cores?